src/phoenix.js

```diff
@@ -0,0 +1,730 @@
+// Phoenix Channels JavaScript client
```

Comment:
Unfortunately the module isn't official and seems out of date (there are some API differences that I'm not sure about). I think including the source is safer for now; I'll ask on the mailing list if there's an "official" npm module.